cmd/sos.go

```diff
@@ -65,6 +66,8 @@ func newSOSClient(certsFile string) (*sosClient, error) {
 		if !c.certPool.AppendCertsFromPEM(certs) {
 			return nil, errors.New("unable to load local certificates")
 		}
+	} else if runtime.GOOS == "windows" {
+		os.Stderr.WriteString("It looks like you are running on Windows and you didn't pass the --certs-file flag. Due to a bug in Golang this may not work.\n")
```

Comment:
Instead of just saying that is doesn't work, this should refer to `exo sos help` for the details and solution.

Reply:
Thank you, I will to that if we don't manage to get #244 through.